src/components/cards/RelationshipsCard.jsx

```diff
@@ -162,42 +162,31 @@ export default function RelationshipsCard({
     [siteSettings],
   );
   const relationshipTableData = useMemo(
-    () => {
-      return map(
+    () =>
+      map(
         relationships,
         relationship => {
-          // keeping this in until we after QA and confirm for certain that this would not be needed (I have a note in the ticket to come back and clean this up)
-          // const selfIndividualMembers = filter(
-          //   get(relationship, 'individual_members'),
-          //   individualMember =>
-          //     individualMember.individual_guid !== individualGuid,
-          //   [],
-          // );
-          // const selfIndividualMember = get(selfIndividualMembers, [
-          //   0,
-          // ]);
-          const nonSelfIndividualMembers = filter(
+          const selfIndividualMember = find(
             get(relationship, 'individual_members'),
             individualMember =>
-              individualMember.individual_guid !== individualGuid,
-            [],
+              individualMember.individual_guid === individualGuid,
           );
-          const nonSelfIndividualMember = get(
-            nonSelfIndividualMembers,
-            [0],
+          const nonSelfIndividualMember = find(
+            get(relationship, 'individual_members'),
```

Comment:
it's important not to omit that third argument (fallback value)
`get(relationship, 'individual_members')` should be `get(relationship, 'individual_members', [])`. otherwise if `relationship.individual_members` does not exist the `.map` call throw an error

Reply:
Done